.env.example

```diff
@@ -1,2 +1,2 @@
-DB_USERNAME=
-DB_PASSWORD=
+DB_USERNAME=postgres
+DB_PASSWORD=1234
```

Comment:
Our environment variables go to the `.env` file, the `.env.example` file is just a helper that can be used when the user first clones the repository and make a copy from it. So that's why we don't need to modify this file.